src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java

```diff
@@ -64,6 +65,8 @@ public static short computeDuplicateScore(final SAMRecord record, final ScoringS
 
     /**
      * Returns the duplicate score computed from the given fragment.
+     * value should be capped by Short.MAX_VALUE since the score from two reads will be
```

Comment:
Capped to `Short.MAX_VALUE / 2`?  Maybe where they are summed it should just use an int?